src/targets/mobile/config.xml

```diff
@@ -105,6 +105,6 @@
     </plugin>
     <plugin name="cordova-blur-app-privacy-screen" spec="https://github.com/lifeofcoding/cordova-blur-app-privacy-screen.git" />
     <engine name="ios" spec="4.5.5" />
-    <plugin name="cordova-universal-links-plugin" spec="https://github.com/walteram/cordova-universal-links-plugin.git" />
+    <plugin name="cordova-universal-links-plugin" spec="https://github.com/Mailbutler/cordova-universal-links-plugin"/>
```

Comment:
There is no .git at the end, is it ok ?

Reply:
Yep. It works better without the `.git`. In fact, if you delete the package.json and json-lock and try to install plugin with a `.git` it'll not works. Even weirder, If you copy/paste the line (with the `.git`) and try to install it from the cli, it'll works... Va comprendre